### ISSUE TITLE
polymer-layout: NoSuchMethodError: method not found: 'getComputedStyle' when used as top level element in template

### ISSUE THREAD

**Issue by Andersmholmgren** [bwu-dart-graveyard/polymer_elements#118]
If a polymer-layout element is used without a wrapping element (e.g. div) then polymer-layout gets an NPE. The js version copes with this fine. I'm not sure if this is supposed to be supported but it works with the JS version hence my query. 

How to reproduce
Add polymer-layout as a direct child of the template as per below

```
<!DOCTYPE html>

<link rel="import" href="packages/polymer_elements/polymer_layout/polymer_layout.html">

<polymer-element name="example-ui">
  <template>
    <polymer-layout vertical></polymer-layout>
    <div flex></div>
  </template>
  <script src="example_ui.dart" type="application/dart"></script>
</polymer-element>
```

The dart editor console displays

```
ExampleUi.created
Exception: The null object does not have a setter 'canvasModel='.

NoSuchMethodError: method not found: 'canvasModel='
Receiver: null
Arguments: [Instance of 'DefaultCanvasModel'] (http://127.0.0.1:52986/client/test/ui_components/canvas_view/example_ui.dart:23)
Uncaught Error: The null object does not have a method 'getComputedStyle'.

NoSuchMethodError: method not found: 'getComputedStyle'
Receiver: null
Arguments: []
Stack Trace: 
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:45)
#1      PolymerLayout.prepare (http://127.0.0.1:52986/client/test/ui_components/canvas_view/packages/polymer_elements/polymer_layout/polymer_layout.dart:152:37)
#2      PolymerLayout.enteredView.<anonymous closure> (http://127.0.0.1:52986/client/test/ui_components/canvas_view/packages/polymer_elements/polymer_layout/polymer_layout.dart:145:19)
#3      dirtyCheckZoneSpec.wrapCallback.<anonymous closure> (package:observe/src/dirty_check.dart:118:15)
#4      _rootRun (dart:async/zone.dart:706)
#5      _ZoneDelegate.run (dart:async/zone.dart:440)
#6      _CustomizedZone.run (dart:async/zone.dart:650)
#7      _BaseZone.runGuarded (dart:async/zone.dart:561)
#8      _BaseZone.bindCallback.<anonymous closure> (dart:async/zone.dart:586)
#9      dirtyCheckZoneSpec.wrapCallback.<anonymous closure> (package:observe/src/dirty_check.dart:118:15)
#10     _rootRun (dart:async/zone.dart:710)
#11     _ZoneDelegate.run (dart:async/zone.dart:440)
#12     _CustomizedZone.run (dart:async/zone.dart:650)
#13     _BaseZone.runGuarded (dart:async/zone.dart:561)
#14     _BaseZone.bindCallback.<anonymous closure> (dart:async/zone.dart:586)
#15     _createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:11)
#16     _Timer._Timer.<anonymous closure> (file:///Volumes/data/b/build/slave/dartium-mac-full-dev/build/src/dart/tools/dom/src/native_DOMImplementation.dart:549)


Exception: The null object does not have a method 'getComputedStyle'.

NoSuchMethodError: method not found: 'getComputedStyle'
Receiver: null
Arguments: []

```

**Comment by zoechi** [comment]
There was a limitation in PolymerDart to access the parent. This caused a lot of troubles when developing polymer-flex-layout. 
Now I investigated this again and it is fixed in PolymerDart (similar as in PolymerJS). 
We should now be able to fix the last issues with flex-layout too!

I published a new release 0.1.2+1

**Comment by Andersmholmgren** [comment]
Thanks for all your help Gunter
